src/error.rs

```diff
@@ -17,7 +17,6 @@ pub enum FindexErr {
     Other(String),
 
     // Findex implementation for FFI
-    #[cfg(feature = "interfaces")]
```

Comment:
Why do you remove these lines ? It does not seem to be related to this PR's topic.

Reply:
I don't really know why my changes requires this but the compiler is unhappy :-D `FindexErr::Callback` is used inside `noisy_fetch_chains` which may be used outside the `interface` feature? I don't know… Maybe something with the removal of `futures` as an optional dependency?

Reply:
It was a problem on the `improve_errors` base branch, fixed!